### PR TITLE
solution

### PR DESCRIPTION
**Pull request by Kozubowicz**

### CODE REVIEW

File: src/checkPassword.test.js [hunk 1]
```diff
@@ -7,13 +7,32 @@ describe(`Function 'checkPassword':`, () => {
     expect(checkPassword).toBeInstanceOf(Function);
   });
 
-  it(`should return boolean`, () => {
+  it(`should return boolean`, () => {});
```
Comment (zibi-dev) [inline src/checkPassword.test.js]: missing test

File: src/checkPassword.test.js [hunk 2]
```diff
@@ -7,13 +7,32 @@ describe(`Function 'checkPassword':`, () => {
...
 
+  it(`should return 'true' for the valid password with 8 characters`, () => {});
```
Comment (zibi-dev) [inline src/checkPassword.test.js]: missing test

File: src/checkPassword.test.js [hunk 3]
```diff
@@ -7,13 +7,32 @@ describe(`Function 'checkPassword':`, () => {
...
+
+  // write more tests here
```
Comment (zibi-dev) [inline src/checkPassword.test.js]: remove comments

File: src/checkPassword.test.js [hunk 4]
```diff
@@ -7,13 +7,32 @@ describe(`Function 'checkPassword':`, () => {
...
+
+  it(`should return 'true' for the valid password`, () => {
+    expect(checkPassword('sam1@Zqwer')).toBeTruthy();
+  });
+
+  it(`should return 'true' for the valid password`, () => {
+    expect(checkPassword('jACK1__lol')).toBeTruthy();
   });
```
Comment (zibi-dev) [inline src/checkPassword.test.js]: It can be merged

**Review by Zibi95 — CHANGES_REQUESTED** [review]
Hey, there are some things to fix.

File: src/checkPassword.test.js [hunk 5]
```diff
@@ -4,20 +4,15 @@ describe(`Function 'checkPassword':`, () => {
...
-  it(`should return boolean`, () => {});
-
-  it(`should return 'true' for the valid password with 8 characters`, () => {});
-
-  // write more tests here
-
-  it(`should return 'true' for the valid password`, () => {
-    expect(checkPassword('sam1@Zqwer')).toBeTruthy();
+  it(`should return boolean`, () => {
+    expect(checkPassword).toBeTruthy();
```
Comment (natalia-klonowska) [inline src/checkPassword.test.js]: you should check if result type is boolean

File: src/checkPassword.test.js [hunk 6]
```diff
@@ -4,20 +4,15 @@ describe(`Function 'checkPassword':`, () => {
   const checkPassword = require('./checkPassword');
 
   it(`should be declared`, () => {
-    expect(checkPassword).toBeInstanceOf(Function);
+    expect(checkPassword).toBeDefined();
   });
 
-  it(`should return boolean`, () => {});
-
-  it(`should return 'true' for the valid password with 8 characters`, () => {});
```
Comment (natalia-klonowska) [inline src/checkPassword.test.js]: you should check password with exactly 8 characters

File: src/checkPassword.test.js [hunk 7]
```diff
@@ -4,16 +4,30 @@ describe(`Function 'checkPassword':`, () => {
...
+  });
 
+  it(`should return 'true' for the valid password`, () => {
+    expect(checkPassword('sam1@Zqwer')).toBeTruthy();
+    expect(checkPassword('jACK1__lol')).toBeTruthy();
   });
 
-  it(`should return 'true' for the valid password with 8 characters`, () => {
+  it(`should return 'false' for password bellow 8 characters`, () => {
+    expect(checkPassword('sam1@Z')).toBeFalsy();
```
Comment (natalia-klonowska) [inline src/checkPassword.test.js]: you should check boundary value so 7 not 6 ;)

**Review by natalia-klonowska — CHANGES_REQUESTED** [review]
ensure that function returns explicit boolean values (`true` or `false`) rather than just any "truthy" or "falsy" values by replacing `toBeTruthy()` with `toBe(true)` and `toBeFalsy()` with `toBe(false)`

**Review by Zibi95 — APPROVED** [review]
LGTM!
